docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -112,6 +116,10 @@ values. Typical usage does not require you to configure them.
 * - `delta.max-partitions-per-writer`
   - Maximum number of partitions per writer.
   - `100`
+* - `delta.idle-writer-min-file-size`
+  - Minimum data written by a single partition writer before it can be
+    considered as 'idle' and could be closed by the engine.
```

Comment:
```
Minimum data written by a single partition writer before it can
     be considered as idle and can be closed by the engine. Equivalent
     session property is `idle_writer_min_file_size`.
```